docs/migration/00d_Lattice_models_with_v0.5.ipynb

```diff
@@ -0,0 +1,466 @@
+    "\n",
+    "Furthermore, the factory methods for `Lattice` objects defined on the `LatticeModel` classes were migrated as follows:\n",
+    "\n",
+    "| Legacy method | New method |\n",
+    "|:--------------|:-----------|\n",
+    "| `FermiHubbardModel.uniform_parameters` | `Lattice.uniform_parameters` |\n",
+    "| `FermiHubbardModel.from_parameters` | `Lattice.from_adjacency_matrix` |\n",
+    "| `IsingModel.uniform_parameters` | `Lattice.uniform_parameters` |\n",
+    "| `IsingModel.from_parameters` | `Lattice.from_adjacency_matrix` |\n",
+    "\n",
+    "### Table of Contents\n",
+    "Concrete examples are provided below. You can use the following links to jump to a specific section:\n",
+    "\n",
+    "- [FermiHubbardModel.uniform_parameters](#FermiHubbardModel.uniform_parameters)\n",
+    "- [FermiHubbardModel.from_parameters](#FermiHubbardModel.from_parameters)\n",
+    "- [IsingModel.uniform_parameters](#IsingModel.uniform_parameters)\n",
+    "- [IsingModel.from_parameters](#IsingModel.from_parameters)\n",
+    "\n",
+    "### Further Resources\n",
+    "Be sure to check out the following tutorials for how to use the new code:\n",
+    "\n",
+    "- [Lattice Models](../tutorials/10_lattice_models.ipynb)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "bf6920b6",
+   "metadata": {},
+   "source": [
+    "## `FermiHubbardModel.uniform_parameters`"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "0bf1663b",
+   "metadata": {},
+   "source": [
+    "### Previously"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "e515413c",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Fermionic Operator\n",
+      "register length=4, number terms=10\n",
+      "  (2+0j) * ( +_0 -_2 )\n",
+      "+ (-2+0j) * ( -_0 +_2 )\n",
+      "+ (4+0j) * ( +_0 -_0 )\n",
+      "+ (4+0j) * ( +_2 -_2 )\n",
+      "+ (2+0j) * ( +_1 -_3 )\n",
+      "+ (-2+0j) * ( -_1 +_3 )\n",
+      "+ (4+0j) * ( +_1 -_1 )\n",
+      "+ (4+0j) * ( +_3 -_3 )\n",
+      "+ (3+0j) * ...\n"
+     ]
+    },
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "<ipython-input-1-b4df7055da5b>:1: NatureDeprecationWarning: The qiskit_nature.problems.second_quantization.electronic package is deprecated as of version 0.5.0 and will be removed no sooner than 3 months after the release. Instead use the qiskit_nature.second_q.problems package.\n",
+      "  from qiskit_nature.problems.second_quantization.lattice.lattices import LineLattice\n",
+      "<ipython-input-1-b4df7055da5b>:1: NatureDeprecationWarning: The qiskit_nature.problems.second_quantization.vibrational package is deprecated as of version 0.5.0 and will be removed no sooner than 3 months after the release. Instead use the qiskit_nature.second_q.problems package.\n",
+      "  from qiskit_nature.problems.second_quantization.lattice.lattices import LineLattice\n"
```

Comment:
I commented on these messages before - here it seems even more weird since it mentions electronic and vibrational but nothing specific to lattices

Reply:
@manoelmarques another (weirder) occurrence of the same "problem" I pinged you about in another thread above. I'll leave this unresolved until we figure out what is going on here